--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -37,13 +37,10 @@
-find_package(OpenCV 4.4)
-   if(NOT OpenCV_FOUND)
-      message(FATAL_ERROR "OpenCV > 4.4 not found.")
-   endif()
+find_package(OpenCV REQUIRED)
 
 MESSAGE("OPENCV VERSION:")
 MESSAGE(${OpenCV_VERSION})
 
 find_package(Eigen3 3.1.0 REQUIRED)
-find_package(Pangolin REQUIRED)
+#find_package(Pangolin REQUIRED)
 find_package(realsense2)
 
 include_directories(
@@ -52,7 +49,7 @@
 ${PROJECT_SOURCE_DIR}/include/CameraModels
 ${PROJECT_SOURCE_DIR}/Thirdparty/Sophus
 ${EIGEN3_INCLUDE_DIR}
-${Pangolin_INCLUDE_DIRS}
+#${Pangolin_INCLUDE_DIRS}
 )
 
 set(CMAKE_LIBRARY_OUTPUT_DIRECTORY ${PROJECT_SOURCE_DIR}/lib)
@@ -65,11 +62,11 @@
 src/LoopClosing.cc
 src/ORBextractor.cc
 src/ORBmatcher.cc
-src/FrameDrawer.cc
+#src/FrameDrawer.cc
 src/Converter.cc
 src/MapPoint.cc
 src/KeyFrame.cc
 src/Atlas.cc
 src/Map.cc
-src/MapDrawer.cc
+#src/MapDrawer.cc
 src/Optimizer.cc
 src/Frame.cc
 src/KeyFrameDatabase.cc
 src/Sim3Solver.cc
-src/Viewer.cc
+#src/Viewer.cc
 src/ImuTypes.cc
 src/G2oTypes.cc
 src/CameraModels/Pinhole.cpp
--- a/src/System.cc
+++ b/src/System.cc
@@ -73,11 +73,11 @@
 
-    mpFrameDrawer = new FrameDrawer(mpAtlas);
-    mpMapDrawer = new MapDrawer(mpAtlas, strSettingsFile);
+    mpFrameDrawer = nullptr; //new FrameDrawer(mpAtlas);
+    mpMapDrawer = nullptr; //new MapDrawer(mpAtlas, strSettingsFile);
 
-    mpViewer = new Viewer(this, mpFrameDrawer, mpMapDrawer, mpTracking, strSettingsFile);
-    mptViewer = new thread(&Viewer::Run, mpViewer);
+    mpViewer = nullptr; //new Viewer(this, mpFrameDrawer, mpMapDrawer, mpTracking, strSettingsFile);
+    //mptViewer = new thread(&Viewer::Run, mpViewer);
 
     mpTracking->SetViewer(mpViewer);
--- a/src/Tracking.cc
+++ b/src/Tracking.cc
@@ -432,1 +432,1 @@
-    mpFrameDrawer->Update(this);
+    //mpFrameDrawer->Update(this);
